### PR TITLE
Feat/sprint 2 1 create component deteil vet page

### PR DESCRIPTION
**Pull request by Samwa-ua**

### CODE REVIEW

**Review by sanderpa — APPROVED** [review]

File: client/src/app/vets/components/vet-animals-type/vet-animals-type.component.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@media(max-width: 770px){
+  
+  .animals-types__cards-set .item{
```
Comment (vladtymo) [inline client/src/app/vets/components/vet-animals-type/vet-animals-type.component.scss]: Fix code formatting (spaces, tabulations) in this file

File: client/src/app/vets/components/vet-services/vet-services.component.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<div class="main_services">
+  <h2>Наші послуги</h2>
+</div>
+<section class="services-section">
```
Comment (vladtymo) [inline client/src/app/vets/components/vet-services/vet-services.component.html:4]: Why don't you load the list of services dynamically from an array?

File: client/src/app/vets/components/veterinarians/veterinarians.component.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<div class="main_veterenarians">
+  <h2>Наші ветеринари</h2>
+</div>
+<section class="veterenarians-section">
```
Comment (vladtymo) [inline client/src/app/vets/components/veterinarians/veterinarians.component.html:4]: The same question as previously...

File: client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.html [hunk 4]
```diff
@@ -1,80 +1,56 @@
...
-        </p>
-      </div>
-    </div>
-  </div>
-  <app-vet-list-card></app-vet-list-card>
-</div>
+  <app-vet-services></app-vet-services>
+
+  <app-veterinarians></app-veterinarians>
+</div>
```
Comment (vladtymo) [inline client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.html]: Add empty line at the end

File: client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.scss [hunk 5]
```diff
@@ -18,58 +24,40 @@
...
 }
-.about_position {
-  font-size: 14px;
-  padding-top: 10px;
+.vets-info-section__info_working-hours-blue {
+  color: #4788b6;
 }
+.vets-info-section__info_working-hours-red {
+  color: #bd2237;
+}
```
Comment (vladtymo) [inline client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.scss]: Add empty line

**Review by DimaMirk — APPROVED** [review]

File: client/src/app/vets/components/vet-services/vet-services.component.html (hunk 2 above)
Comment (Samwa-ua) [reply client/src/app/vets/components/vet-services/vet-services.component.html:4]: It wasn`t my task, it is an independent task that other team member was assigned

File: client/src/app/vets/components/veterinarians/veterinarians.component.html (hunk 3 above)
Comment (Samwa-ua) [reply client/src/app/vets/components/veterinarians/veterinarians.component.html:4]: It wasn`t my task, it is an independent task that other team member was assigned

File: client/src/app/vets/components/vet-animals-type/vet-animals-type.component.scss (hunk 1 above)
Comment (Samwa-ua) [reply client/src/app/vets/components/vet-animals-type/vet-animals-type.component.scss]: Fixed with Prettier

File: client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.html (hunk 4 above)
Comment (Samwa-ua) [reply client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.html]: Added empty string at the end

File: client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.scss (hunk 5 above)
Comment (Samwa-ua) [reply client/src/app/vets/pages/vet-unit-page/vet-unit-page.component.scss]: Added empty string at the end

**Pull request merged**